stream/src/main/scala/org/apache/pekko/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -62,6 +62,8 @@ import pekko.util.OptionVal
 
   val Debug = false
 
+  val Mailbox: String = "pekko.stream.materializer.mailbox"
```

Comment:
can you call this param 'MailboxConfigName' ?

Reply:
Can be done.